Elemental/Components/AeCheckbox/AeCheckbox.razor

```diff
@@ -15,7 +15,7 @@
     }
     @if (!HasValue)
     {
-        @if (CheckedIcon == null)
+        @if (NullIcon == null)
```

Comment:
I wasn't aware there was a bug with the checkbox - does it still handle the three states?

Reply:
It still handles the three states **IF** CheckedIcon == null.
It seemed like it was a mistake to check for CheckedIcon == null when deciding which icon to use for NullIcon, so I changed it to NullIcon == null.
If it was working as intended you can change it back.

Reply:
I'm not sure about this, @nicksalt maybe you can chime in?